# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1483742485 28800
#      Fri Jan 06 14:41:25 2017 -0800
# Node ID 62eb83755336e696bc7081503bbe9776e8586631
# Parent  624f76f2ca512992e7b09d3d3ee74c7c32167da8
configure: reword SUNDIALS IDAKLU warning message for clarity

* configure.ac: Reword warning message when missing SUNDIALS with IDAKLU solver
support for clarity.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -1555,17 +1555,17 @@ OCTAVE_CHECK_LIB(klu, KLU,
   [], [don't use KLU library, disable some sparse matrix functionality])
 
 ### Check for SUNDIALS_IDA library and header.
 save_CPPFLAGS="$CPPFLAGS"
 save_LIBS="$LIBS"
 CPPFLAGS="$KLU_CPPFLAGS $CPPFLAGS"
 LIBS="$KLU_LDFLAGS $KLU_LIBS $LIBS"
 OCTAVE_CHECK_LIB(sundials_ida, SUNDIALS_IDA,
-  [SUNDIALS_IDA library configured with klu not found.  Solvers ode15i and ode15s will be disabled.],
+  [SUNDIALS_IDA library with IDAKLU solver support not found.  Solvers ode15i and ode15s will be disabled.],
   [ida/ida.h ida.h], [IDAKLU], [], [],
   [warn_sundials_ida=
    OCTAVE_CHECK_SIZEOF_SUNDIALS_IDA_REALTYPE])
 LIBS="$save_LIBS"
 CPPFLAGS="$save_CPPFLAGS"
 
 ## Additional check for sundials_nvecserial library
 OCTAVE_CHECK_LIB(sundials_nvecserial, SUNDIALS_NVECSERIAL,
